.concourse/concourse.yaml

```diff
@@ -132,22 +132,28 @@ jobs:
           args:
           - -ec
           - |
+            # see github issue #65, so we take an older version that works with 1.1
+            cd istio-installer
+            git fetch origin master
+            git checkout 02d6d54442fb1f48528742a40a37c3a37888db79
```

Comment:
HEAD should (and absolutely, P0) needs to work with release-1.1. 

What happens when you use the latest, we should be able to install alongside an existing version with no interference?

Reply:
> HEAD should (and absolutely, P0) needs to work with release-1.1.



Just to clarify: You mean that **upgrading** from an installation of release-1.1 should work with the current installer, correct? Installing 1.1 with the new installer is not supported according to [this issue](https://github.com/istio/istio/issues/14146).

> What happens when you use the latest, we should be able to install alongside an existing version with no interference?

 
When we use the latest version of the new installer to upgrade from 1.1, the error that @c0d1ngm0nk3y mentioned occurs. To install alongside an existing version, we would have to modify this pipeline to use a canary-style upgrade (and not in-place) for all other components as well (so far, only istio-control is upgraded like this, i.e. by installing separately + switching).